tuxedo_sysfs/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "tuxedo_sysfs"
 authors = ["Aaron Erhardt <aaron.erhardt@t-online.de>"]
-version = "0.2.0"
+version = "0.2.3"
```

Comment:
I think using `version.workspace = true` might be a good idea here: [example from the cargo book](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table). Unless we plan to assign individual versions, this would simply save a lot of work. The same goes for the `edition`, `license` and `rust-version` keys.

Reply:
That sounds like it might simplify things :)

I'm still playing around with the release-please action in my fork.

Reply:
Well, I've hit the rate limit for testing.
Had [a successful run](https://github.com/mrcjkb/tuxedo-rs/actions/runs/6713320510) though, so I think this is ready to be merged.